### ISSUE TITLE
Accept filename and code block id as parameter

### ISSUE THREAD

**Issue by christian-bromann** [stateful/runme#82]
Suggest to remove the `--chdir` and `filename` parameter as it feels not very familiar. Given the natural way devs us `ls` is suggest to allow passing in the readme path as parameter, e.g.:

Instead of:
```sh
runme ls --chdir ./examples --filename README.md
```

Just do:

```sh
runme ls ./examples/README.md
```

Same for executing code blocks:

```sh
runme run ./examples/README.md myExampleCodeBlock
```

**Comment by adambabik** [comment]
I agree that `--chdir` and `--filename` are a poor UX design choice. The suggestions also make sense. The only thing is to provide good defaults which seem to fit nicely here:

* `runme ls` would default to the current working directory and `README.md`
* `runme run myExampleCodeBlock` would work the same and it's easy to distinct these two examples by the number of provided arguments.

**Comment by christian-bromann** [comment]
@adambabik makes sense to me
